src/scenes/home/footer/footer.css

```diff
@@ -163,18 +164,10 @@
   .blockRow a {
     padding: 10px 10px;
   }
-
-  .footer {
-    height: 650px;
-  }
 }
 
 @media screen and (max-width: 480px) {
   .blockGroup a {
     padding-left: 3rem;
   }
-
-  .footer {
-    padding-bottom: 50px;
-  }
 }
```

Comment:
@Jacfem In light of Conrad's clarifying comment that the CivicX image should be above the footer (not below it), it would probably be best  to not change any of the footer CSS for this issue, since changes to the footer won't be needed.

Reply:
updated!